vendor.conf

```diff
@@ -11,14 +11,13 @@ github.com/pmezard/go-difflib master
 github.com/pkg/errors master
 golang.org/x/crypto/openpgp master
 # docker deps from https://github.com/docker/docker/blob/v1.11.2/hack/vendor.sh
-github.com/docker/docker v1.12.1
-github.com/docker/engine-api 4eca04ae18f4f93f40196a17b9aa6e11262a7269
+github.com/docker/docker v1.13.0
 github.com/docker/go-connections 4ccf312bf1d35e5dbda654e57a9be4c3f3cd0366
-github.com/vbatts/tar-split v0.9.11
+github.com/vbatts/tar-split v0.10.1
 github.com/gorilla/context 14f550f51a
 github.com/gorilla/mux e444e69cbd
-github.com/docker/go-units 651fc226e7441360384da338d0fd37f2440ffbe3
-golang.org/x/net master https://github.com/golang/net.git
+github.com/docker/go-units 8a7beacffa3009a9ac66bad506b18ffdd110cf97
+golang.org/x/net 2beffdc2e92c8a3027590f898fe88f69af48a3f8 https://github.com/tonistiigi/net.git
```

Comment:
… I am being stupid; of course that the fork only _changes_ `x/net/trace` does not mean that the two have no diverged in the intervening 9 months.